v2/cmd/wails/internal/commands/dev/dev.go

```diff
@@ -33,36 +33,14 @@ import (
 
 	"github.com/fsnotify/fsnotify"
 	"github.com/leaanthony/clir"
+	commonFlags "github.com/wailsapp/wails/v2/cmd/wails/internal/flags"
 	"github.com/wailsapp/wails/v2/internal/fs"
 	"github.com/wailsapp/wails/v2/internal/process"
 	"github.com/wailsapp/wails/v2/pkg/clilogger"
 	"github.com/wailsapp/wails/v2/pkg/commands/build"
+	"github.com/wailsapp/wails/v2/cmd/wails/internal/utils"
```

Comment:
In Go, you generally don't have ambiguous package names. In this instance you may want to call it "logutils" or something. It actually doesn't bother me so much, more an FYI 👍

Reply:
great tip, thank you! will refactor it and keep it in mind!